CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to rubi-py
+
+Thank you for investing your time in contributing to rubi! In this guide, you will get a quick overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR
+
+## New contributor guide
+
+To get an overview of rubi, please read about the [Rubicon Protocol](), the [README](https://github.com/RubiconDeFi/rubi-py/blob/master/rubi/README.md) and our [docs][]. These will get you up to speed and help you get started with this repository
+
+## Get started 
+
+#### Create an Issues
+
+If you spot an issue with our codebase, first search to see if the issue already exists in the `Issues` tab. If a related issue does not exist, please open an issue and fill out our template to be as detailed as possible. Our team will get to it as soon as possible!
+
+#### Contributions
+
+Coming soon!
```

Comment:
Maybe worth making an open issue to add details here

Reply:
Will do